scripts/deployment/deploy.sh

```diff
@@ -35,15 +36,20 @@ fi
 case $opt in
         auto_deploy)
             chmod 400 scripts/deployment/evalai.pem
-            ssh ubuntu@${HOSTNAME} -i scripts/deployment/evalai.pem -o StrictHostKeyChecking=no \
-            "cd ~/Projects/evalai && \
-            export AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID} && \
-            export COMMIT_ID=${COMMIT_ID} && \
-            eval $(aws ecr get-login --no-include-email) && \
-            aws s3 cp s3://cloudcv-secrets/evalai/${env}/docker_${env}.env ./docker/prod/docker_${env}.env && \
-            docker-compose -f docker-compose-${env}.yml rm -s -v -f && \
-            docker-compose -f docker-compose-${env}.yml pull && \
-            docker-compose -f docker-compose-${env}.yml up -d --force-recreate --remove-orphans django nodejs nodejs_v2 celery "
+            ssh-add scripts/deployment/evalai.pem
+			ssh -A ubuntu@${JUMPBOX} -o StrictHostKeyChecking=no INSTANCE=${INSTANCE} AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID} COMMIT_ID=${COMMIT_ID} env=${env} 'bash -s' <<-'ENDSSH'
```

Comment:
these commands have to be indented using tabs, here block doesn't work with spaces. You might see misaligned code on git.